boa_engine/src/syntax/parser/expression/unary.rs

```diff
@@ -120,6 +120,12 @@ where
                 cursor.next(interner)?.expect("! token vanished"); // Consume the token.
                 Ok(node::UnaryOp::new(UnaryOp::Not, self.parse(cursor, interner)?).into())
             }
+            TokenKind::Keyword((Keyword::Await, true)) if self.allow_await.0 => Err(
+                ParseError::general("Keyword must not contain escaped characters", token_start),
```

Comment:
I saw that this error is not taken into account for other keywords. Is that exclusive to `await` (meaning it would be good to change the error message) or is that a missing error in other keywords?

Reply:
We also have this error for other keywords so you are right, we can make the error more specific here.